website/events/models/event.py

```diff
@@ -179,6 +179,10 @@ class Event(models.Model, metaclass=ModelTranslateMeta):
         null=True,
     )
 
+    tpay_allowed = models.BooleanField(
+        _("Is Thalia Pay payment allowed for this event"), default=True
```

Comment:
![image](https://user-images.githubusercontent.com/1799914/108087498-ce549180-7077-11eb-94f2-7b89819b4c9e.png)
This text looks a bit weird :p

Reply:
Yeah that indeed looks weird.
I think `Allow Thalia Pay payments` would be better. We could also go for `Allow Thalia Payments` but I'm not sure if we want to use that terminology (maybe @JobDoesburg can answer this)

Reply:
I think the first one is best. Since 'Thalia Pay' is kind of the 'thing'.

Reply:
Does that make sense? I don't know :p

Reply:
Or just "Allow Thalia Pay"